update-lb4-docs.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const srcDocs = path.resolve(
+  __dirname,
+  'node_modules',
+  '@loopback',
+  'docs',
+  'site',
+);
+const srcImages = path.resolve(srcDocs, 'imgs');
+const destDocs = path.resolve(__dirname, 'pages', 'en', 'lb4');
+const destImages = path.resolve(__dirname, 'images', 'lb4');
+
+async function copyDocs(src, dest) {
+  try {
+    await fs.copy(src, dest);
+  } catch (err) {
+    console.error('failed to copy docs %s', err.stack);
+  }
+}
+
+copyDocs(srcDocs, destDocs);
+copyDocs(srcImages, destImages);
+fs
+  .ensureDir(path.resolve(destDocs, 'imgs'))
+  .then(() => {
+    fs.removeSync(path.resolve(destDocs, 'imgs'));
```

Comment:
No need to check existence of `imgs` when we use `fs.removeSync` (rm -rf). See https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove-sync.md.